templates/books-show.html

```diff
@@ -0,0 +1,47 @@
+<div>
+  <div class="col-md-10 col-md-offset-1">
```

Comment:
Same issue with so many outer divs that don't seem to do anything. This still feels like old, dead, broken code.